sections/conclusion/conclusion-chapter.tex

```diff
@@ -1 +1,8 @@
 \section{Conclusion}\label{sec:conclusion}
+Throughout this paper, we have looked into various metrics used for evaluating recommender systems. What we found was that there is no consistent set of metrics used for evaluation, but the most commonly used are NDCG and Recall.
+We found that utilizing bipartite graphs is a popular and intuitive way to handle information in the system, but in the investigated metrics, they are often outperformed by traditional methods such as SVD.
```

Comment:
snakker vi RS i stedet for system? og snakker vi side-information?

Reply:
og skal vi spåecifre det er for graf baserede metoder?

Reply:
mangler viu ikke også noget omkring context i de her start sætninger

Reply:
> 
> 
> tænker også der mangler noget konkret såsom at den her metode er bedst på den her metric

Yes, det er også bare første outcast - der er en del der mangler her i endnu